strawberry/subscriptions/protocols/graphql_transport_ws/handlers.py

```diff
@@ -167,7 +167,8 @@ async def handle_connection_init(self, message: ConnectionInitMessage) -> None:
             self.connection_init_timeout_task.cancel()
 
         payload = message.get("payload", {})
-
+        if payload is None:
```

Comment:
Not breaking for graphql_transport_ws, because it was required to be dict.

It's possible to make other way around and allow null in graphql_transport_ws, like in graphql_ws. 
But since it's "recommended" protocol I decided to change graphql_ws and not the other way around.